docs/sources/helm-charts/mimir-distributed/configure/configure-gem-federation-frontend.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Configure GEM federation-frontend"
+menuTitle: "GEM federation-frontend"
+description: "Learn how to enable the GEM federation-frontend."
+---
+
+# Configure GEM federation-frontend
+
+You can deploy the Grafana Enterprise Metrics (GEM) federation-frontend in a Kubernetes cluster using Helm. The federation-frontend allows you to query data from multiple GEM clusters using a single endpoint. For more information about cluster query federation, refer to the [federation-frontend documentation](https://grafana.com/docs/enterprise-metrics/<GEM_VERSION>/operations/cluster-query-federation).
+
+{{< admonition type="note" >}}
+This guide focuses specifically on deploying the federation-frontend component as a standalone deployment without any additional GEM or Mimir components.
+{{< /admonition >}}
+
+## Before you begin
+
+1. Set up a GEM cluster: For information about setting up and configuring a GEM deployment, refer to the [Configure Grafana Enterprise Metrics]({{< relref "./configure-grafana-enterprise-metrics" >}}) and [Get started with Grafana Mimir using the Helm chart]({{< relref "../get-started-helm-charts" >}}) articles.
```

Comment:
```suggestion
1. Set up a GEM cluster: For information about setting up and configuring a GEM deployment, refer to [Configure Grafana Enterprise Metrics]({{< relref "./configure-grafana-enterprise-metrics" >}}) and [Get started with Grafana Mimir using the Helm chart]({{< relref "../get-started-helm-charts" >}}).
```

Reply:
We switched to using full URLs and not relrefs for links, per https://grafana.com/docs/writers-toolkit/write/links/#link-to-grafanacom-pages.

Reply:
the cascade variable for the mimir-distributed docs isn't present and the rest of the helm docs use relrefs. I'll stick with relrefs for now